dataclients/kubernetes/kube.go

```diff
@@ -279,14 +317,49 @@ func (c *Client) loadAndConvert() ([]*eskip.Route, error) {
 	return r, nil
 }
 
+func healthcheckRoute(healthy bool) *eskip.Route {
+	status := http.StatusOK
+	if !healthy {
+		status = http.StatusNotFound
```

Comment:
Yes, 5xx is more appropriate! (does not make a practical difference, but we should follow HTTP standards..)